pkg/tcl/README.md

```diff
@@ -0,0 +1,7 @@
+# Testkube Operator - TCL Package
+
+This folder contains special code with the Testkube Community license.
+
+## License
+
+The code in this folder is licensed under the Testkube Community License. Please see the [LICENSE](../../licenses/TCL.txt) file for more information.
```

Comment:
we should add a link to the FAQ here is as well - but it could be in a separate PR if this needs to be merged..

Reply:
Added a link to the FAQ as well given how these two PRs will likely be merged around the same time.